front/app/components/UI/FullscreenModal/index.tsx

```diff
@@ -136,71 +123,10 @@ class FullscreenModal extends PureComponent<Props, State> {
       });
   }
 
-  componentDidUpdate(prevProps: Props) {
-    if (!prevProps.opened && this.props.opened) {
-      this.openModal();
-    } else if (prevProps.opened && !this.props.opened) {
-      this.cleanup();
-    }
-  }
-
   componentWillUnmount() {
     this.subscription?.unsubscribe();
-    this.cleanup();
   }
 
-  openModal = () => {
-    const { locale, url, goBackUrl } = this.props;
-
-    if (!isNilOrError(locale) && url) {
-      const { pathname } = removeLocale(url);
-      this.url = `${window.location.origin}/${locale}${pathname}`;
-      this.goBackUrl = `${window.location.origin}/${locale}${
-        removeLocale(goBackUrl || window.location.pathname).pathname
-      }`;
-      window.history.pushState({ path: this.url }, '', this.url);
-      window.addEventListener('popstate', this.handlePopstateEvent, useCapture);
-      window.addEventListener('keydown', this.handleKeypress, useCapture);
-      this.unlisten = clHistory.listen(() => this.props.close());
-
-      trackPage(this.url, { modal: true });
-    }
-  };
-
-  handleKeypress = (event: KeyboardEvent) => {
-    if (event.type === 'keydown' && event.key === 'Escape') {
-      event.preventDefault();
-      this.props.close();
-    }
-  };
-
-  handlePopstateEvent = () => {
-    this.props.close();
-  };
-
-  cleanup = () => {
```

Comment:
Since `openModal` was never successful, nothing that's being removed here was ever set, so it can be removed as well.